src/sentry/static/sentry/app/views/settings/components/forms/formPanel.tsx

```diff
@@ -20,7 +20,7 @@ type Props = {
 
   // TODO(ts): See if this is still in use
   access?: Scope[];
-  features?: string[];
+  features?: {[key: string]: any};
```

Comment:
I added a generic for this kind of type recently `StringMap<T>` for these kinds of types.

Reply:
oh I didn't know about StringMap.. thanks for the tip